qt6-qtbase.yaml

```diff
@@ -122,5 +122,7 @@ subpackages:
 
 update:
   enabled: true
-  release-monitor:
-    identifier: 7927
+  github:
+    identifier: qt/qtbase
+    use-tag: true
+    strip-prefix: v
```

Comment:
Can we ignore rc and beta versions as well

Reply:
``
ignore-regex-patterns:
    - 'rc\d+$'
    - 'beta\d+$'
```